apps/condo/pages/property/[id]/index.tsx

```diff
@@ -82,25 +96,40 @@ export const PropertyPageContent = ({ property, role }) => {
             {
                 role && role.canManageProperties ? (
                     <ActionBar>
-                        <Link href={`/property/${property.id}/update`}>
-                            <span>
-                                <Button
-                                    color={'green'}
-                                    type={'sberPrimary'}
-                                    secondary
-                                    icon={<EditFilled />}
-                                    size={'large'}
-                                >
-                                    {UpdateTitle}
-                                </Button>
-                            </span>
-                        </Link>
-                        <DeleteButtonWithConfirmModal
-                            title={ConfirmDeleteTitle}
-                            message={ConfirmDeleteMessage}
-                            okButtonLabel={DeletePropertyLabel}
-                            action={() => softDeleteAction({}, property)}
-                        />
+                        <Space size={20}>
+                            <Link href={`/property/${property.id}/update`}>
+                                <span>
+                                    <Button
+                                        type={'sberDefaultGradient'}
+                                        size={'large'}
+                                    >
+                                        {EditPropertyTitle}
+                                    </Button>
+                                </span>
+                            </Link>
+                            {
+                                property.map !== null && (
+                                    <Link href={`/property/${property.id}/map/update`}>
+                                        <Button
+                                            color={'green'}
+                                            type={'sberDefaultGradient'}
+                                            secondary
+                                            size={'large'}
+                                        >
+                                            {EditPropertyMapTitle}
+                                        </Button>
+                                    </Link>
+                                )
+                            }
+                            <DeleteButtonWithConfirmModal
+                                title={ConfirmDeleteTitle}
+                                message={ConfirmDeleteMessage}
+                                okButtonLabel={DeletePropertyLabel}
+                                action={() => softDeleteAction({}, property)}
```

Comment:
useCallback

Reply:
@akarjakin can't use useCallback here because action prop requires method that return `Promise` as result of execution

Reply:
It doesn't look obvious. Looks like we have naming problem. Maybe it'd be better to add ticket for refactoring such naming issues, to make them more predictable